packages/devtools-evm/src/omnigraph/sdk.ts

```diff
@@ -44,6 +44,7 @@ export abstract class OmniSDK implements IOmniSDK {
 
     constructor(
         public readonly contract: OmniContract,
+        public readonly contractName: string = '',
```

Comment:
@janjakubnanista adding this so that I can pass the contract name when calling the constructor [here](https://github.com/stargate-protocol/stargate-v2/blob/ff015c3b75968f155e62ff79d49a813fedc9819d/packages/stg-devtools-evm-hardhat-v2/src/asset/factory.ts#L17) for example

Reply:
ravina todo: test if `this.contractName` is accessible like `this.contract` is

Reply:
remove this change